### PR TITLE
Maikksmt patch 1

### PR DESCRIPTION
**Pull request by maikksmt**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by jens1o** [comment]
Stop it. I want to contribute something too :D

(sarcasm)

File: _articles/de-DE/starting-a-project.md [hunk 1]
```diff
@@ -1,302 +1,300 @@
 ---
-locale: en-US
-title: Starting an Open Source Project
-description: Learn more about the world of open source and get ready to launch your own project.
-class: beginners
+locale: de-DE
+title: Ein Open-Source-Projekt starten
+description: Erfahre mehr über die Welt von Open Source und mache dich bereit, dein eigenes Projekt zu starten.
+class: Anfänger
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: leave it as `beginners`, it is not publicly visible, but instead it is used as a css class, which modifies some style for some pages.

File: _articles/de-DE/starting-a-project.md [hunk 2]
```diff
@@ -1,302 +1,300 @@
...
 toc:
-  the-what-and-why-of-open-source: "The what and why of open source"
-  should-i-launch-my-own-open-source-project: "Should I launch my own open source project?"
-  launching-your-own-open-source-project: "Launching your own open source project"
-  naming-and-branding-your-project: "Naming and branding your project"
-  your-pre-launch-checklist: "Your pre-launch checklist"
+  das-was-und-warum-von-open-source: "as "Was" und "Warum" von Open-Source"
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: only translate the values in the `""`.

`Das`

**Comment by jens1o** [comment]
will review later in detail, as it is too much for this time :D

File: _articles/de-DE/starting-a-project.md [hunk 3]
```diff
@@ -1,302 +1,301 @@
...
+description: Erfahre mehr über die Welt von Open Source und mache dich bereit, dein eigenes Projekt zu starten.
 class: beginners
+
 toc:
-  the-what-and-why-of-open-source: "The what and why of open source"
-  should-i-launch-my-own-open-source-project: "Should I launch my own open source project?"
-  launching-your-own-open-source-project: "Launching your own open source project"
-  naming-and-branding-your-project: "Naming and branding your project"
-  your-pre-launch-checklist: "Your pre-launch checklist"
+  the-what-and-why-of-open-source: "Das "Was" und "Warum" von Open-Source"
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: I'm not sure, but I think the extra `"` should be either escaped or removed. Toss a coin ;)

File: _articles/de-DE/starting-a-project.md [hunk 4]
```diff
@@ -1,302 +1,301 @@
...
 related:
   - finding
   - building
 ---
 
-## The "what" and "why" of open source
+## Das "Was" und "Warum" von Open-Source
 
-So you're thinking about getting started with open source? Congratulations! The world appreciates your contribution. Let's talk about what open source is and why people do it.
+Du denkst also darüber nach, mit Open Source zu beginnen? Glückwünsche! Die Welt schätzt deinen Beitrag. Lass uns darüber reden, was Open Source ist und warum es Leute tun.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `Open-Source` :D

File: _articles/de-DE/starting-a-project.md [hunk 5]
```diff
@@ -1,302 +1,301 @@
...
 
-### What does "open source" mean?
+### Was bedeutet "Open-Source"?
 
-When a project is open source, that means **anybody can view, use, modify, and distribute your project for any purpose.** These permissions are enforced through [an open source license](https://opensource.org/licenses).
+Wenn ein Projekt Open-Source ist, bedeutet das: **Jeder kann Ihr Projekt zu einem beliebigen Zweck anzeigen, verwenden, ändern und verteilen.** Diese Berechtigungen werden durch [eine Open-Source-Lizenz](https://opensource.org/licenses) erzwungen.
 
-Open source is powerful because it lowers the barriers to adoption, allowing ideas to spread quickly.
+Open Source ist mächtig, weil es die Hürden für die Einführung senkt und Ideen schnell verbreiten kann.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `Open-Source`

File: _articles/de-DE/starting-a-project.md [hunk 6]
```diff
@@ -1,302 +1,301 @@
...
-### Why do people open source their work?
+### Warum machen Menschen ihre Arbeit Open-Source?
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars1.githubusercontent.com/u/1500684?v=3&s=460" class="pquote-avatar" alt="avatar">
-  One of the most rewarding experiences I get out of using and collaborating on open source comes from the relationships that I build with other developers facing many of the same problems I am.
+  Eine der lohnendsten Erfahrungen, die ich bei der Verwendung und Zusammenarbeit mit Open-Source machen kann, sind die Beziehungen, die ich mit anderen Entwicklern aufbaue, die mit den gleichen Problemen konfrontiert sind wie ich.
   <p markdown="1" class="pquote-credit">
-— @kentcdodds, ["How getting into Open Source has been awesome for me"](https://medium.com/@kentcdodds/how-getting-into-open-source-has-been-awesome-for-me-8480cd756a80#.pjt9oqp4w)
+— @kentcdodds, ["Wie ich zu Open-Source kam, war für mich großartig"](https://medium.com/@kentcdodds/how-getting-into-open-source-has-been-awesome-for-me-8480cd756a80#.pjt9oqp4w)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: Could you append ` (English)` to the link title, please?

File: _articles/de-DE/starting-a-project.md [hunk 7]
```diff
@@ -1,302 +1,301 @@
...
   </p>
 </aside>
 
-[There are many reasons](http://ben.balter.com/2015/11/23/why-open-source/) why a person or organization would want to open source a project. Some examples include:
-
-* **Collaboration:** Open source projects can accept changes from anybody in the world. [Exercism](https://github.com/exercism/), for example, is a programming exercise platform with over 350 contributors.
+[Es gibt viele Gründe](http://ben.balter.com/2015/11/23/why-open-source/) warum eine Person oder Organisation ein Open-Source-Projekt starten möchte. Einige Beispiele sind:
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 8]
```diff
@@ -1,302 +1,301 @@
...
 
-* **Adoption and remixing:** Open source projects can be used by anyone for nearly any purpose. People can even use it to build other things. [WordPress](https://github.com/WordPress), for example, started as a fork of an existing project called [b2](https://github.com/WordPress/book/blob/master/Content/Part%201/2-b2-cafelog.md).
+* **Zusammenarbeit:** Open-Source-Projekte können Änderungen von jedem auf der Welt akzeptieren. [Exercism] (https://github.com/exercism/) ist zum Beispiel eine Übungsplattform mit über 350 Teilnehmern.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 9]
```diff
@@ -1,302 +1,301 @@
...
 
-Open source isn't just for software, either. You can open source everything from data sets to books. Check out [GitHub Explore](https://github.com/explore) for ideas on what else you can open source.
+* **Transparenz:** Jeder kann ein Open-Source-Projekt auf Fehler oder Inkonsistenzen untersuchen. Transparenz ist für Regierungen wie [Bulgarien](https://medium.com/@bozhobg/bulgaria-got-a-law-requiring-open-source-98bf626cf70a) wichtig oder die [Vereinigten Staaten](https://sourcecode.cio.gov/), regulierte Branchen wie Banken oder das Gesundheitswesen und Sicherheitssoftware wie [Let's Encrypt](https://github.com/letsencrypt).
 
-### Does open source mean "free of charge"?
+Open-Source ist nicht nur für Software. Du kannst alles, von Datensätzen bis Bücher, Open-Source machen. Schaue dir [GitHub Explore](https://github.com/explore) an, um zu erfahren, was du sonst noch alles tun kannst.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 10]
```diff
@@ -1,302 +1,301 @@
...
+### Bedeutet Open-Source "kostenlos"?
 
-Because [an open source license requires](https://opensource.org/osd-annotated) that anyone can use, modify, and share your project for nearly any purpose, projects themselves tend to be free of charge. If the project cost money to use, anyone could legally make a copy and use the free version instead.
+Einer der größten Vorteile von Open-Source ist, dass es kein Geld kostet. "Kostenlos" ist jedoch ein Nebenprodukt des Gesamtwertes von Open-Source.
 
-As a result, most open source projects are free, but "free of charge" is not part of the open source definition. There are ways to charge for open source projects indirectly through dual licensing or limited features, while still complying with the official definition of open source.
+Da [eine Open-Source-Lizenz erfordert](https://opensource.org/osd-annotated), dass jeder Ihr Projekt für fast jeden Zweck nutzen, ändern und teilen kann, sind Projekte in der Regel kostenlos. Wenn das Projekt Geld kostet, kann jeder legal eine Kopie machen und stattdessen die kostenlose Version verwenden.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 11]
```diff
@@ -1,302 +1,301 @@
...
+ Irgendwann erstellte ich eine benutzerdefinierte UIAlertView, die ich verwendete ... und ich entschied mich dafür, es Open-Source zu machen. Also habe ich es etwas dynamischer gestaltet und in GitHub hochgeladen. Ich habe auch meine erste Dokumentation geschrieben, die anderen Entwicklern erklärt, wie man sie in ihren Projekten verwendet. Wahrscheinlich hat es niemand benutzt, weil es ein einfaches Projekt war, aber ich fühlte mich gut mit meinem Beitrag.
   <p markdown="1" class="pquote-credit">
-— @mavris, ["Self-taught Software Developers: Why Open Source is important to us"](https://medium.com/rocknnull/self-taught-software-engineers-why-open-source-is-important-to-us-fe2a3473a576#.zhwo5krlq)
+— @mavris, ["Autodidaktische Softwareentwickler: Warum Open-Source für uns wichtig ist"](https://medium.com/rocknnull/self-taught-software-engineers-why-open-source-is-important-to-us-fe2a3473a576#.zhwo5krlq)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 12]
```diff
@@ -1,302 +1,301 @@
...
   <img src="https://avatars2.githubusercontent.com/u/1857993?v=3&s=460" class="pquote-avatar" alt="avatar">
-  As you begin to open source the project, it's important to make sure that your management processes take into consideration the contributions and abilities of the community around your project. Don't be afraid to involve contributors who are not employed in your business in key aspects of the project — especially if they are frequent contributors.
+  Wenn du beginnst, das Open-Source-Projekt zu starten, musst du sicherstellen, dass deine Verwaltungsprozesse die Beiträge und Fähigkeiten der Gemeinschaft rund um dein Projekt berücksichtigen. Scheue dich nicht, Mitwirkende, die in deinem Unternehmen nicht angestellt sind, in Schlüsselaspekte des Projekts mit einzubeziehen - insbesondere, wenn sie häufig Beiträge leisten.
   <p markdown="1" class="pquote-credit">
-— @captainsafia, ["So you wanna open source a project, eh?"](https://writing.safia.rocks/2016/12/06/so-you-wanna-open-source-a-project-eh/)
+— @captainsafia, ["Du willst also ein Open-Source-Projekt starten, oder?"](https://writing.safia.rocks/2016/12/06/so-you-wanna-open-source-a-project-eh/)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 13]
```diff
@@ -1,302 +1,301 @@
...
+Es gibt keine perfekte Zeit, um deine Arbeit Open-Source zu machen. Du kannst eine Idee, eine in Bearbeitung befindliche Arbeit oder ein jahrelang geschlossenes Projekt Open-Source machen.
 
-Generally speaking, you should open source your project when you feel comfortable having others view, and give feedback on, your work.
+Im Allgemeinen solltest du dein Projekt Open-Source machen, wenn du dich damit zufrieden gibst, dass andere deine Arbeit sehen und Feedback geben.
 
-No matter which stage you decide to open source your project, every project should include the following documentation:
+Unabhängig davon, in welcher Phase du dich entscheidest, dein Projekt Open-Source zu machen, sollte jedes Projekt folgende Dokumentation enthalten:
 
-* [Open source license](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository)
+* [Open-Source Lizenz](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `Open-Source-Lizenz (Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 14]
```diff
@@ -1,302 +1,301 @@
...
 * [README](https://help.github.com/articles/create-a-repo/#commit-your-first-change)
-* [Contributing guidelines](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)
-* [Code of conduct](../code-of-conduct/)
+* [Beitragsrichtlinien](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)
+* [Verhaltenskodex](../code-of-conduct/)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 15]
```diff
@@ -1,302 +1,301 @@
...
 
-No matter which stage you decide to open source your project, every project should include the following documentation:
+Unabhängig davon, in welcher Phase du dich entscheidest, dein Projekt Open-Source zu machen, sollte jedes Projekt folgende Dokumentation enthalten:
 
-* [Open source license](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository)
+* [Open-Source Lizenz](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository)
 * [README](https://help.github.com/articles/create-a-repo/#commit-your-first-change)
-* [Contributing guidelines](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)
-* [Code of conduct](../code-of-conduct/)
+* [Beitragsrichtlinien](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 16]
```diff
@@ -1,302 +1,301 @@
...
+![Wähle eine Lizenz aus](/assets/images/starting-a-project/repository-license-picker.png)
 
-If you have other questions or concerns around the legal aspects of managing an open source project, [we've got you covered](../legal/).
+Wenn du andere Fragen oder Bedenken bezüglich der rechtlichen Aspekte der Verwaltung eines Open-Source-Projekts hast, [Geben wir dir Deckung](../legal/).
 
-### Writing a README
+### Eine README schreiben
 
-READMEs do more than explain how to use your project. They also explain why your project matters, and what your users can do with it.
+READMEs beinhalten mehr als die Erklärung, wie dein Projekt Verwendet wird. Sie erklären auch, warum dein Projekt wichtig ist und was die Benutzer damit machen können.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: I think `Verwendet` needs to be lowercased, doesn't it?

File: _articles/de-DE/starting-a-project.md [hunk 17]
```diff
@@ -1,302 +1,301 @@
...
 
-In your README, try to answer the following questions:
+Versuche in Ihrer README die folgenden Fragen zu beantworten:
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `deiner`

File: _articles/de-DE/starting-a-project.md [hunk 18]
```diff
@@ -1,302 +1,301 @@
...
+Du kannst deine README verwenden, um weitere Fragen zu beantworten, z. B. wie du mit Beiträgen umgehst, was die Ziele des Projekts sind und Informationen zu Lizenzen und Zuordnungen. Wenn du keine Beiträge annehmen möchtest oder dein Projekt noch nicht für den produktiven Einsatz bereit ist, schreibe diese Informationen nieder.
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars0.githubusercontent.com/u/168572?v=3&s=460" class="pquote-avatar" alt="avatar">
-  Better documentation means more users, less support requests, and more contributors. (...) Remember that your readers aren't you. There are people who might come to a project who have completely different experiences.
+  Eine bessere Dokumentation bedeutet mehr Nutzer, weniger Supportanfragen und mehr Mitwirkende. (...) Denke daran, dass deine Leser nicht wie du sind. Es gibt Leute, die vielleicht zu einem Projekt kommen, die ganz andere Erfahrungen haben.
   <p markdown="1" class="pquote-credit">
 — @limedaring, ["Writing So Your Words Are Read (video)"](https://www.youtube.com/watch?v=8LiV759Bje0&list=PLmV2D6sIiX3U03qc-FPXgLFGFkccCEtfv&index=10)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: Could you translate this too, like you did before? (And append `(Englisch)`?)

File: _articles/de-DE/starting-a-project.md [hunk 19]
```diff
@@ -1,302 +1,301 @@
...
   </p>
 </aside>
 
-Sometimes, people avoid writing a README because they feel like the project is unfinished, or they don't want contributions. These are all very good reasons to write one.
+Manchmal vermeiden es Leute eine README zu schreiben, weil sie das Gefühl haben, dass das Projekt unvollendet ist, oder sie wollen keine Beiträge. Das sind alles gute Gründe, eine README zu schreiben.
 
-For more inspiration, try using @18F's ["Making READMEs Readable"](https://pages.18f.gov/open-source-guide/making-readmes-readable/) or @PurpleBooth's [README template](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) to write a complete README.
+Für weitere Anregungen kannst du @18F's ["READMEs lesbar machen"](https://pages.18f.gov/open-source-guide/making-readmes-readable/) oder @PurpleBooth's [README-Vorlage](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) verwenden um eine komplette README zu schreiben.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)` 2x

File: _articles/de-DE/starting-a-project.md [hunk 20]
```diff
@@ -1,302 +1,301 @@
...
-### Writing your contributing guidelines
+### Schreibe deine Beitragsrichtlinien
 
-A CONTRIBUTING file tells your audience how to participate in your project. For example, you might include information on:
+Eine CONTRIBUTING-Datei teilt deinem Publikum mit, wie Sie an deinem Projekt teilnehmen können. Beispielsweise könntest du Informationen zu folgenden Themen hinzufügen:
 
-* How to file a bug report (try using [issue and pull request templates](https://github.com/blog/2111-issue-and-pull-request-templates))
-* How to suggest a new feature
-* How to set up your environment and run tests
+* Wie man einen Fehlerbericht einreicht (Versuch es mit [Issue- und Pull-Request-Vorlagen](https://github.com/blog/2111-issue-and-pull-request-templates))
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 21]
```diff
@@ -1,302 +1,301 @@
...
-* How contributors should (or should not) get in touch with you
+* Welche Arten von Beiträgen du suchst
+* Deinen Fahrplan oder deine Vision für das Projekt
+* Wie Mitwirkende mit dir in Kontakt treten sollten (oder nicht sollten)
 
-Using a warm, friendly tone and offering specific suggestions for contributions (such as writing documentation, or making a website) can go a long way in making newcomers feel welcomed and excited to participate.
+Ein warmer, freundlicher Tonfall und das Angebot spezifischer Vorschläge für Beiträge (wie das Schreiben von Dokumentation oder das Erstellen einer Website) können dazu beitragen, dass Neulinge sich willkommen und begeistert fühlen.
 
-For example, [Active Admin](https://github.com/activeadmin/activeadmin/) starts [its contributing guide](https://github.com/activeadmin/activeadmin/blob/master/CONTRIBUTING.md) with:
+Beispiel: [Active Admin](https://github.com/activeadmin/activeadmin/) startet [den dazugehörigen Leitfaden](https://github.com/activeadmin/activeadmin/blob/master/CONTRIBUTING.md) mit:
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 22]
```diff
@@ -1,302 +1,301 @@
...
 
-> First off, thank you for considering contributing to Active Admin. It's people like you that make Active Admin such a great tool.
+> Zunächst einmal vielen Dank, dass Sie erwägt haben, zum Active Admin beizutragen. Es sind Leute wie Sie, die Active Admin zu einem großartigen Tool machen.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: informal forms, please. OSS Development is most of the time informal. :)

File: _articles/de-DE/starting-a-project.md [hunk 23]
```diff
@@ -1,302 +1,301 @@
...
 
-In the earliest stages of your project, your CONTRIBUTING file can be simple. You should always explain how to report bugs or file issues, and any technical requirements (like tests) to make a contribution.
+In den ersten Phasen deines Projekts kann Ihre CONTRIBUTING-Datei einfach sein. Du solltest immer erklären, wie Fehler oder Probleme mit Dateien gemeldet werden können und welche technischen Anforderungen (z.B. Tests) erforderlich sind, um einen Beitrag zu leisten.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: informal

File: _articles/de-DE/starting-a-project.md [hunk 24]
```diff
@@ -1,302 +1,301 @@
...
 
-Over time, you might add other frequently asked questions to your CONTRIBUTING file. Writing down this information means fewer people will ask you the same questions over and over again.
+Mit der Zeit kannst du deiner CONTRIBUTING-Datei andere häufig gestellte Fragen hinzufügen. Wenn du diese Informationen notierst, werden immer weniger Menschen dieselben Fragen immer wieder stellen.
 
-For more help with writing your CONTRIBUTING file, check out @nayafia's [contributing guide template](https://github.com/nayafia/contributing-template/blob/master/CONTRIBUTING-template.md) or @mozilla's ["How to Build a CONTRIBUTING.md"](http://mozillascience.github.io/working-open-workshop/contributing/).
+Weitere Hilfe beim Schreiben deiner CONTRIBUTING-Datei findest du in @nayafia's [Leitfaden-Vorlage](https://github.com/nayafia/contributing-template/blob/master/CONTRIBUTING-template.md) oder @mozilla's ["Wie erstellt man eine CONTRIBUTING.md"](http://mozillascience.github.io/working-open-workshop/contributing/).
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 25]
```diff
@@ -1,302 +1,301 @@
...
+— @mlynch, ["Open-Source zu einem glücklicheren Ort machen"](https://medium.com/ionic-and-the-mobile-web/making-open-source-a-happier-place-3b90d254f5f#.v4qhl7t7v)
   </p>
 </aside>
 
-Finally, a code of conduct helps set ground rules for behavior for your project's participants. This is especially valuable if you're launching an open source project for a community or company. A code of conduct empowers you to facilitate healthy, constructive community behavior, which will reduce your stress as a maintainer.
-
-For more information, check out our [Code of Conduct guide](../code-of-conduct/).
+Schließlich hilft ein Verhaltenskodex, Verhaltensregeln für die Teilnehmer Ihres Projekts festzulegen. Dies ist besonders nützlich, wenn du ein Open-Source-Projekt für eine Gemeinschaft oder ein Unternehmen startest. Ein Verhaltenskodex ermächtigt dich, ein gesundes, konstruktives Gemeinschaftsverhalten zu fördern, welches deinen Stress als Projektleiter reduziert.
+Weitere Informationen findest du in unserem [Verhaltenskodex](../code-of-conduct/).
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: Could you readd the line before this paragraph?

File: _articles/de-DE/starting-a-project.md [hunk 26]
```diff
@@ -1,302 +1,301 @@
...
+Zusätzlich zur Kommunikation darüber, _wie_ du erwartest, dass sich die Teilnehmer verhalten, beschreibt ein Verhaltenskodex auch, auf wen sich diese Erwartungen beziehen, wann sie gelten und was zu tun ist, wenn eine Verletzung dessen auftritt.
 
-Much like open source licenses, there are also emerging standards for codes of conduct, so you don't have to write your own. The [Contributor Covenant](http://contributor-covenant.org/) is a drop-in code of conduct that is used by [over 40,000 open source projects](http://contributor-covenant.org/adopters/), including Kubernetes, Rails, and Swift. No matter which text you use, you should be prepared to enforce your code of conduct when necessary.
+Ähnlich wie bei Open-Source-Lizenzen gibt es auch neue Standards für Verhaltenskodizes, sodass du keine eigenen schreiben musst. Der [Mitwirkenden-Bund](http://contributor-covenant.org/) ist ein Drop-In-Verhaltenskodex, der von [über 40.000 Open-Source-Projekten](http://contributor-covenant.org/adopters/) verwendet wird, einschließlich Kubernetes, Rails, and Swift. Unabhängig davon, welchen Text du verwendest, solltest du bereit sein, deinen Verhaltenskodex bei Bedarf durchzusetzen.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 27]
```diff
@@ -1,302 +1,301 @@
...
+Betrachte vor allem Klarheit. Wortspiele machen Spaß, aber denke daran, dass einige Witze nicht zu anderen Kulturen oder Menschen mit unterschiedlichen Erfahrungen von dir übersetzt werden können. Einige deiner potenziellen Benutzer könnten Mitarbeiter eines Unternehmens sein: Du möchtest sie nicht in Verlegenheit bringen, indem du ihnen dein Projekt bei der Arbeit erklären musst!
 
-### Avoiding name conflicts
+### Namenskonflikte vermeiden
 
-[Check for open source projects with a similar name](http://ivantomic.com/projects/ospnc/), especially if you share the same language or ecosystem. If your name overlaps with a popular existing project, you might confuse your audience.
+[Suche nach Open-Source-Projekten mit ähnlichem Namen](http://ivantomic.com/projects/ospnc/), insbesondere wenn du dieselbe Sprache oder dasselbe Ökosystem verwendest. Wenn sich dein Name mit einem bekannten vorhandenen Projekt überschneidet, könntest du deine Zielgruppe verwirren.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 28]
```diff
@@ -1,302 +1,301 @@
...
 
-If you want a website, Twitter handle, or other properties to represent your project, make sure you can get the names you want. Ideally, [reserve those names now](https://instantdomainsearch.com/) for peace of mind, even if you don't intend to use them just yet.
+Wenn du eine Website, einen Twitter-Konto  oder andere Eigenschaften zur Darstellung deines Projekts wünschst, stelle sicher, dass du die gewünschten Namen erhalten kannst. Im Idealfall [reserviere diese Namen jetzt](https://instantdomainsearch.com/), damit du beruhigt sein kannst, selbst wenn du sie noch nicht verwenden willst.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 29]
```diff
@@ -1,302 +1,301 @@
...
 
-You can check the [WIPO Global Brand Database](http://www.wipo.int/branddb/en/) for trademark conflicts. If you're at a company, this is one of the things your [legal team can help you with](../legal/).
+Du kannst die [WIPO Globale Markendatenbank](http://www.wipo.int/branddb/en/) auf Markenrechtskonflikte prüfen. Wenn du in einem Unternehmen bist, ist dies eines der Dinge, mit denen dein [juristisches Team dir helfen kann](../legal/).
 
-Finally, do a quick Google search for your project name. Will people be able to easily find your project? Does something else appear in the search results that you wouldn't want them to see?
+Zum Schluss führe eine schnelle Google-Suche nach deinem Projektnamen durch. Werden die Leute dein Projekt leicht finden können? Erscheint etwas anderes in den Suchergebnissen, das Sie nicht sehen sollen?
 
-### How you write (and code) affects your brand, too!
+### Wie du schreibst (und Programmierst) wirkt sich auf die Marke aus!
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `Programmierst` should be lowercased, shouldn't it?

File: _articles/de-DE/starting-a-project.md [hunk 30]
```diff
@@ -1,302 +1,301 @@
...
   <img src="https://avatars0.githubusercontent.com/u/11321?v=3&s=460" class="pquote-avatar" alt="avatar">
-  I tried to be involved with every thread on the mailing list, and showing exemplary behaviour, being nice to people, taking their issues seriously and trying to be helpful overall. After a while, people stuck around not to only ask questions, but to help with the answering as well, and to my complete delight, they mimicked my style.
+  Ich habe versucht, mich in jeden Thread auf der Mailingliste einzubringen und vorbildliches Verhalten zu zeigen, nett zu Menschen zu sein, ihre Probleme ernst zu nehmen und insgesamt hilfreich zu sein. Nach einer Weile hielten sich die Leute daran, nicht nur Fragen zu stellen, sondern auch bei der Beantwortung zu helfen, und zu meiner vollen Freude ahmten sie meinen Stil nach.
   <p markdown="1" class="pquote-credit">
-— @janl on [CouchDB](https://github.com/apache/couchdb), ["Sustainable Open Source"](http://writing.jan.io/2015/11/20/sustainable-open-source.html)
+— @janl on [CouchDB](https://github.com/apache/couchdb), ["Nachhaltiges Open-Source"](http://writing.jan.io/2015/11/20/sustainable-open-source.html)
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 31]
```diff
@@ -1,302 +1,301 @@
...
   </p>
 </aside>
 
-Using warm, inclusive language (such as "them", even when referring to the single person) can go a long way in making your project feel welcoming to new contributors. Stick to simple language, as many of your readers may not be native English speakers.
+Wenn du eine freundliche, willkommene Formulierung verwendest (wie "ihr", auch wenn du sich auf eine einzelne Person beziehst), kann dies dazu beitragen, dass sich neue Mitwirkende bei deinem Projekt wohlfühlen. Halte dich an einfache Formulierungen, da viele deiner Leser nicht Muttersprachler sind.
 
-Beyond how you write words, your coding style may also become part of your project's brand. [Angular](https://github.com/johnpapa/angular-styleguide) and [jQuery](http://contribute.jquery.org/style-guide/js/) are two examples of projects with rigorous coding styles and guidelines.
+Über die Art und Weise, wie du Texte schreibst, kann auch dein Programmierstil Teil der Marke deines Projekts werden. [Angular](https://github.com/johnpapa/angular-styleguide) und [jQuery](http://contribute.jquery.org/style-guide/js/) sind zwei Beispiele für Projekte mit strengen Programmierstilen und -richtlinien.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md [hunk 32]
```diff
@@ -1,302 +1,301 @@
...
 
-It isn't necessary to write a style guide for your project when you're just starting out, and you may find that you enjoy incorporating different coding styles into your project anyway. But you should anticipate how your writing and coding style might attract or discourage different types of people. The earliest stages of your project are your opportunity to set the precedent you wish to see.
+Es ist nicht notwendig, eine Gestaltungsrichtlinnie für dein Projekt zu schreiben, wenndu gerade erst anfängst, und du wirst vielleicht feststellen, dass es dir Spaß macht, verschiedene Programmierstile in dein Projekt zu integrieren. Aber du solltest vorhersehen, wie dein Schreib- und Progammierstil verschiedene Arten von Menschen anziehen oder entmutigen könnte. Die frühesten Phasen deines Projekts sind deine Chance, den Präzedenzfall zu setzen, den du sehen möchtest.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: missing space between `wenn` and `du`

File: _articles/de-DE/starting-a-project.md [hunk 33]
```diff
@@ -1,302 +1,301 @@
...
 
-## Your pre-launch checklist
+## Deine Checkliste vor dem Start
 
-Ready to open source your project? Here's a checklist to help. Check all the boxes? You're ready to go! [Click "publish"](https://help.github.com/articles/making-a-private-repository-public/) and pat yourself on the back.
+Bist du bereit, dein Projekt Open-Source zu machen? Hier ist eine Checkliste die dir hilft. Hast du alle Punkte erledigt? Du bist bereit loszulegen! [Klicke auf "Veröffentlichen"](https://help.github.com/articles/making-a-private-repository-public/) und klopfe dir auf die Schulter.
```
Comment (jens1o) [inline _articles/de-DE/starting-a-project.md]: `(Englisch)`

File: _articles/de-DE/starting-a-project.md (hunk 8 above)
Comment (maikksmt) [reply _articles/de-DE/starting-a-project.md]: This is a link to a project, not an English guide or article. Should that still be marked as English?

File: _articles/de-DE/starting-a-project.md (hunk 22 above)
Comment (maikksmt) [reply _articles/de-DE/starting-a-project.md]: what is OSS?

File: _articles/de-DE/starting-a-project.md (hunk 8 above)
Comment (jens1o) [reply _articles/de-DE/starting-a-project.md]: err... After thinking, I think it's already fine :)

**Pull request merged**